app/models/concerns/vacancy_job_summary_validations.rb

```diff
@@ -0,0 +1,9 @@
+module VacancyJobSummaryValidations
```

Comment:
Overall I'm dubious about adding an additional class just to have two validations in two different models (one of which,`JobSummaryForm`, isn't really necessary either). Leave it for now, but let's start to think about getting rid of some of this layered-model-class complexity.